build.gradle.kts

```diff
@@ -178,6 +193,60 @@ group = "com.virtuslab"
 
 configureVersionFromGit()
 
+repositories {
+  mavenCentral()
+  intellijPlatform {
+    defaultRepositories()
+  }
+}
+
+intellijPlatform {
+  buildSearchableOptions = false
+  instrumentCode = false
+  pluginConfiguration {
+    name = "git-machete"
+    description = file("$rootDir/DESCRIPTION.html").readText()
+    ideaVersion {
+      sinceBuild = IntellijVersionHelper.versionToBuildNumber(intellijVersions.earliestSupportedMajor)
+      untilBuild = IntellijVersionHelper.versionToBuildNumber(intellijVersions.latestSupportedMajor) + ".*"
+    }
+  }
+  signing {
+    certificateChain = pluginSignCertificateChain?.trimIndent()
+    privateKey = pluginSignPrivateKey?.trimIndent()
+    password = pluginSignPrivateKeyPass
+  }
+  verifyPlugin {
```

Comment:
This task doesn't work yet (checks only a single version, or some random subset of version)